.github/workflows/ci.yaml

```diff
@@ -31,6 +31,9 @@ jobs:
           check-latest: true
           cache: 'npm'
       - run: npm ci
+      - name: install website dependencies
+        working-directory: website
+        run: npm ci
```

Comment:
I have the CI running things like `typecheck` on the website folder.  There is code that ends up relying on dependent packages for types so the typechecking was failing without doing this.

I'd like to work on switching this to a true monorepo format where the website is just another package which theoretically would allow a single `npm install` to install for all package folders.